eth_defi/aave_v3/rates.py

```diff
@@ -0,0 +1,174 @@
+"""Aave v3 rate calculation."""
+import logging
+from datetime import datetime
+from decimal import Decimal
+from typing import Tuple
+
+from pandas import DataFrame, Timedelta
+
+logger = logging.getLogger(__name__)
+
+# Constants for APY and APR calculation
+RAY = Decimal(10**27)  # 10 to the power 27
+WAD = Decimal(10**18)  # 10 to the power 18
+SECONDS_PER_YEAR_INT = 31536000
```

Comment:
I suggest writing this literal value with a thousands separator, easier to parse when glancing over it.